src/file.ts

```diff
@@ -1120,6 +1121,10 @@ class File extends ServiceObject<File> {
    *     NOTE: Byte ranges are inclusive; that is, `options.start = 0` and
    *     `options.end = 999` represent the first 1000 bytes in a file or object.
    *     NOTE: when specifying a byte range, data integrity is not available.
+   * @property {boolean} [decompress=true] Disable auto decompression of the
+   *     received data. By default this option set to `true`.
```

Comment:
this option *is* set to `true`

Reply:
Currently nodejs-storage client _does_ decompress by default if data received from the server is compressed 
i. e. response contains header `content-encoding: 'gzip'`

`decompress` property seemed a better sounding name vs originally offered `returnCompressed` as per suggestion [above](https://github.com/googleapis/nodejs-storage/pull/850#discussion_r323419992)

Reply:
There is a missing "is" in the sentence: "By default this option set to `true`."

Reply:
> There is a missing "is" in the sentence: "By default this option set to `true`."

fixed, thanks